wfe/wfe.go

```diff
@@ -232,34 +235,107 @@ func (wfe *WebFrontEndImpl) sendError(prob *acme.ProblemDetails, response http.R
 	_, _ = response.Write(problemDoc)
 }
 
+func getAlternateNo(url string) (string, int, int) {
```

Comment:
I think this function is a little bit sparse for comments. I'm not entirely sure I follow the URL parsing logic as-is or what the functions return values are.

Reply:
I've added comments. I hope it's clearer now.